L1Trigger/CSCTrackFinder/test/BuildFile.xml

```diff
@@ -1,51 +1,51 @@
-<use   name="SimDataFormats/Track"/>
```

Comment:
@fwyzard

Reply:
reformatted the comments to use the proper XML syntax, and sorted